2017-11-28-Enhancing+Your+iOS+Application+With+Salmove+SDK.md

```diff
@@ -95,6 +95,13 @@ Here is an example how a controller could be implemented:
 
 ```swift
 extension UIViewController: Interactable {
+    var onScreenSharingOffer: ScreenshareOfferBlock {
+        // When the Operator asks for sharing the screen, supply the answer
```

Comment:
I think we want to show visitor some view asking for permission and then accept or decline the request.
Something like:
```
showScreenSharingRequestView { accepted in
  answer(true)
}
```
No need to implement `showScreenSharingRequestView` tho.

Reply:
Also fix it in this file.